### ISSUE TITLE
Order of includes statements is not a well-defined concept

### ISSUE THREAD

**Issue by bzbarsky** [heycam/webidl#473]
https://heycam.github.io/webidl/#idl-interface-mixins says:

> The order of appearance of includes statements affects the order in which interface mixin are included by their host interface.

But "order of appearance of includes statements" is not really defined, since they can appear in multiple specifications.

As a data point, what Gecko does in practice right now (pre-mixin, in the "implements" and consequential interface world), is to sort the consequential interfaces by name, then include their members in that order.

**Comment by tobie** [comment]
Yeah, this is what I was trying to get at here: https://github.com/heycam/webidl/issues/432.
